src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/MessageReadinessConsumerTest.java

```diff
@@ -43,45 +39,9 @@ void should_not_modify_message_state_if_no_case_event_messages_returned_from_db(
         verify(caseEventMessageRepository, never()).updateMessageState(any(), any());
     }
 
-    @Test
-    void should_not_modify_message_state_if_launch_darkly_feature_flag_is_disabled() {
-        final CaseEventMessageEntity caseEventMessageEntity = TestFixtures.createCaseEventMessageEntity();
-
-        when(caseEventMessageRepository.getAllMessagesInNewState())
-                .thenReturn(List.of(caseEventMessageEntity));
-
-        when(launchDarklyFeatureFlagProvider
-                .getBooleanValue(FeatureFlag.DLQ_DB_PROCESS, TestFixtures.USER_ID))
-                .thenReturn(false);
-
-        messageReadinessConsumer.run();
-
-        verify(caseEventMessageRepository, never()).updateMessageState(any(), any());
-    }
-
-    @Test
-    void should_not_modify_message_state_if_launch_darkly_feature_flag_is_disabled_because_no_user_id_exists() {
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.